api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```
@@ -10156,88 +11210,2472 @@ paths:
+  /admins:
+    get:
+      summary: List Admins
+      tags:
+        - admins
+      responses:
+        '200':
+          $ref: '#/components/responses/admins-response'
+      operationId: get-admins
+      description: Returns a list of admins. The `status` field in the response indicates the state of the admins invitation. `0`= Approved, `1`= Pending, `2`= Rejected, `3`= Revoked, `4` = Invited, `5`= Unverified.
+    post:
+      summary: Invite an Admin
+      operationId: post-admins
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  admin:
+                    type: object
+                    properties:
+                      created_at:
+                        type: integer
+                      id:
+                        type: string
+                      updated_at:
+                        type: integer
+                      status:
+                        type: integer
+                      username:
+                        type: string
+                      email:
+                        type: string
+                      rbac_token_enabled:
+                        type: boolean
+                x-examples:
+                  Example 1:
+                    admin:
+                      created_at: 1556638641
+                      id: 8f0a742f-07f3-49e0-90d7-4fc7eea7e6a4
+                      updated_at: 1556638641
+                      status: 4
+                      username: test-case-3
+                      email: test3@test.com
+                      rbac_token_enabled: true
+              examples:
+                Example admin response:
+                  value:
+                    admin:
+                      created_at: 0
+                      id: string
+                      updated_at: 0
+                      status: 0
+                      username: string
+                      email: string
+                      rbac_token_enabled: true
+      tags:
+        - admins
+      description: Invite an admin to your organization.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                email:
+                  type: string
+                  example: email@example.com
+                  description: The Admin's email address.
+                username:
+                  type: string
+                  description: The Admin's username
+                  example: myusername
+                custom_id:
+                  type: string
+                  description: The Admin's custom ID
+                rbac_token_enabled:
+                  type: boolean
+                  description: Allows the Admin to use and reset their RBAC token; `true` by default
```

```suggestion
                  description: Allows the admin to use and reset their RBAC token.
```
The next line lists the default value.